### ISSUE TITLE
Sync: Add option to sync only selected types of saved sessions

### ISSUE THREAD

**Issue by ovgolovin** [sienori/Tab-Session-Manager#560]
It has been several times that Chrome lost my opened tabs and I wasn't able to restore them.

One of "Tab Session Manager" use cases for me is to auto save sessions as often as possible, so that I can restore it next time when my computer hangs and Chrome fails to restore open tabs.

![image](https://user-images.githubusercontent.com/1408822/84615444-e38c7700-ae7d-11ea-923a-09626ba3a7d5.png)

The other use case is to store about 20 "User sessions" which I use on regular basis with each of them having predefined set of tabs I need to work on particular task.

With current implementation it syncs all sessions including autosaved sessions.
And there are too many of them to sync each time:

![image](https://user-images.githubusercontent.com/1408822/84615638-67466380-ae7e-11ea-99ae-3d2540344df1.png)

Could you add a setting to sync only selected session types, e.g. "User saved" sessions? This will let me keep "Auto Save" sessions only local.

**Comment by walkercam** [comment]
While this doesn't help with your actual request, if all you are worried about is your most recent browsing activity when saving sessions regularly why do you need to save 10,000 sessions? Surely 5/10/60 sessions at one minute intervals would be enough to safeguard your session data in the event of a Chrome meltdown.

**Comment by ovgolovin** [comment]
@walkercam Yes, you are right. To recover from a crash I need to save only the last session.
The reason why I chose 10000 is because sometimes I accidentally close the tabs which I actually needed, only to find this happened hours after. This is why I set it to 10K, to be able to move back in time to previous session state and find that tab (browser history may solve this as well, but not as convenient as all my sessions are mixed into 1 line history there).
